lib/jekyll-import/importers/mt.rb

```diff
@@ -58,15 +65,25 @@ def self.specify_options(c)
       def self.process(options)
         options  = default_options.merge(options)
 
+        engine   = options.fetch('engine')
         dbname   = options.fetch('dbname')
         user     = options.fetch('user')
         pass     = options.fetch('password', "")
         host     = options.fetch('host', "localhost")
+        port     = options.fetch('port')
         comments = options.fetch('comments')
 
         posts_name_by_id = { } if comments
 
-        db = Sequel.mysql(dbname, :user => user, :password => pass, :host => host)
+        if engine == 'mysql'
+          db = Sequel.mysql(dbname, :user => user, :password => pass, :host => host, :port => port )
+        elsif engine == 'postgres'
+          db = Sequel.postgres(dbname, :user => user, :password => pass, :host => host, :port => port )
```

Comment:
the arguments to these two Sequel calls are the same. You could always do `Sequel.public_send(engine, dbname, opts)` if you wanted